sites/partners/pages/listings/applications/add.tsx

```diff
@@ -1,8 +1,9 @@
 import React from "react"
 import Head from "next/head"
-import { MetaTags, PageHeader, SiteAlert, t } from "@bloom-housing/ui-components"
```

Comment:
Why was MetaTags removed from ui-components?

Reply:
MetaTags was using the `next/head` dependency, so in order to remove Next from the component package we'll need to handle anything in the HTML `<head>` through Next in the site folders.